src/main/java/org/cactoos/list/LengthOf.java

```diff
@@ -44,9 +45,17 @@
 
     /**
      * Ctor.
+     * @param items The array
+     */
+    public LengthOf(final Iterable<?> items) {
+        this(items.iterator());
```

Comment:
@yegor256 BTW how about these lines
https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/text/Sprintf.java#L59
and
https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/text/Sprintf.java#L69
Here is code in ctors :smile: 

I guess you have to clearly define what kind of code is acceptable in ctors in addition of simple assignments.